brokers/auth/consts.go

```diff
@@ -0,0 +1,25 @@
+// Package auth contains the authentication related code.
+package auth
```

Comment:
What do you think about moving the the `auth` and `layout` packages from `brokers/` to `pkg/`? I find it a bit odd that these packages are below a `brokers` directory, because the way I see it, these are authd's authentication modes and layouts, not the brokers' - the brokers only choose which of those modes and layouts they support.

Putting packages meant to be imported by other projects below `pkg/` is a common pattern in Go projects and IMO brings some structure into our already quite chaotic project structure.

Reply:
> Well, that issue contains a lengthy discussion, was never resolved and the project layout still contains [pkg](https://github.com/golang-standards/project-layout/tree/master/pkg), so it's just not true that there is consensus on that.Well, that issue contains a lengthy discussion, was never resolved and the project layout still contains [pkg](https://github.com/golang-standards/project-layout/tree/master/pkg), so it's just not true that there is consensus on that.

Nor than "Putting packages meant to be imported by other projects below pkg/ is a common pattern in Go projects " is true nowday.

To move this forward, what about just moving them to the root of authd? How many do we envision having and do they need separate namespacing?

Reply:
To be solved in the follow-up: https://warthogs.atlassian.net/browse/UDENG-5522